appcelerator/titanium_mobile
# Android: WebView null exception fix

**JIRA:**
https://jira.appcelerator.org/browse/TIMOB-26993

**Note:**
This issue was sometimes causing random crashes in our mocha unit test suite below. This solves that issue. And a new unit test is not needed.
https://github.com/appcelerator/titanium-mobile-mocha-suite/blob/master/Resources/ti.ui.webview.test.js

**Test:**
1. Build and run the below code on Android.
2. Tap on the "Open Child" button.
3. Verify that the `WebView` displays Google's website zoomed-in.
4. Tap the Android "Back" button.
5. Tap on the "Open Child" button again.
6. Verify that the `WebView` displays Google's website zoomed-in again.

```javascript
var parentWindow = Ti.UI.createWindow();
var openButton = Ti.UI.createButton({ title: "Open Child" });
openButton.addEventListener("click", function() {
	var childWindow = Ti.UI.createWindow({ backgroundColor: "white" });
	var webView = Ti.UI.createWebView({
		url: "https://www.google.com",
	});
	webView.addEventListener("load", function() {
		webView.zoomLevel = 3.0;
		childWindow.close();
	});
	childWindow.add(webView);
	childWindow.open();
});
parentWindow.add(openButton);
parentWindow.open();
```
